src/components/batch-models-multi-select/CopyInUserBatchModelsMultiSelect.component.js

```diff
@@ -105,25 +101,20 @@ export default class CopyInUserBatchModelsMultiSelectComponent extends React.Com
     }
 
     renderStrategyToggle() {
-        if (this.state.parents && this.state.parents.length > 1) {
-            const label =
-                this.getTranslation("update_strategy") +
-                ": " +
-                this.getTranslation("update_strategy_" + this.state.updateStrategy);
-
-            return (
-                <Toggle
-                    label={label}
-                    style={this.styles.toggle}
-                    checked={this.state.updateStrategy === "replace"}
-                    onToggle={(ev, newValue) =>
-                        this.setState({ updateStrategy: newValue ? "replace" : "merge" })
-                    }
-                />
-            );
-        } else {
-            return null;
-        }
+        const label =
+            this.getTranslation("update_strategy") +
+            ": " +
+            this.getTranslation("update_strategy_" + this.state.updateStrategy);
+        return (
+            <Toggle
+                label={label}
+                style={{ width: 280, float: "right", marginTop: 20, marginRight: 15 }}
```

Comment:
[minor, no need to change] As a rule of thumb, we use immutable props. For styles, that means using the syles object with a reference here.